code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -204,8 +204,13 @@
 		/obj/item/flashlight/pen,
 		/obj/item/seeds,
 		/obj/item/wirecutters,
+		/obj/item/wrench,
+   		/obj/item/reagent_containers/spray/weedspray,
+    	/obj/item/reagent_containers/spray/pestspray,
         /obj/item/wrench,
-	)
+        /obj/item/reagent_containers/spray/weedspray,
+        /obj/item/reagent_containers/spray/pestspray
+        )
```

Comment:
The indents here seems really funky. Did you mix tabs and spaces?

Reply:
oh lordy. did that trying to fix a merge conflict, aagh

Reply:
fixing now